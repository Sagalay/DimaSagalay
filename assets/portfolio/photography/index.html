<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Photography</title>
    <link rel="stylesheet" href="assets/style/style.css" />
  </head>
  <body>
    <!-- Header -->

    <header class="header" id="header">
      <div class="container">
        <div class="header__wrapper">
          <div class="logo">logo</div>
          <nav class="nav" id="nav">
            <a href="#">Home</a>
            <a href="#">About us</a>
            <a href="#">Services</a>
            <a href="#">Team</a>
            <a href="#">Contact us</a>
          </nav>
          <button class="burger" id="navToggle" type="button">
            <span class="burger__item">Menu</span>
          </button>
        </div>
      </div>
    </header>

    <!-- Intro -->
    <div class="intro" id="intro">
      <div class="intro__filter">
        <div class="container">
          <div class="intro__wrapper">
            <h1 class="intro__title">Photography Web templates</h1>
            <a class="button" href="#">
              Read more
            </a>
          </div>
        </div>
      </div>
    </div>

    <!-- About Us -->
    <div class="about">
      <div class="container">
        <div class="about__wrapper">
          <div class="about__img">
            <img src="assets/img/about.jpg" alt="" />
          </div>
          <div class="about__content">
            <h2 class="about__title">About Us</h2>
            <p class="about__subtitle">
              This is Photoshop's version of Lorem Ipsum. Proin gravida nibh vel
              velit auctor aliquet. Aenean sollicitudin, lorem quis bibendum
              auctor, nisi elit
            </p>
            <p class="about__subtitle">
              consequat ipsum, nec sagittis sem nibh id elit. Duis sed odio sit
              amet nibh vulputate cursus a sit amet mauris. Morbi accumsan ipsum
              velit. Nam nec tellus a odio tincidunt auctor a ornare odio.
            </p>
            <div class="button__wrapper">
              <a class="button about__button" href="#">
                Read more
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>

    <!-- Portfolio -->
    <div class="portfolio">
      <div class="container">
        <div class="portfolio__text">
          <div class="portfolio__title">
            Portfolio
          </div>
          <div class="portfolio__subtitle">
            nec sagittis sem nibh id elit. Duis sed odio sit amet nibh vulputate
            cursus a sit amet mauris. Morbi accumsan ipsum velit. Nam nec tellus
            a odio tincidunt auctor a ornare odio.
          </div>
        </div>
        <div class="portfolio__item">
          <div class="portfolio__col">
            <img
              class="portfolio__img"
              src="assets/img/portfolio1.jpg"
              alt=""
            />
            <img
              class="portfolio__img"
              src="assets/img/portfolio2.jpg"
              alt=""
            />
          </div>
          <div class="portfolio__col">
            <img
              class="portfolio__img"
              src="assets/img/portfolio3.jpg"
              alt=""
            />
            <img
              class="portfolio__img"
              src="assets/img/portfolio4.jpg"
              alt=""
            />
          </div>
        </div>
      </div>
    </div>

    <!-- Footer -->
    <footer class="footer">
      <div class="container">
        <div class="contact">
          <div class="contact__wrapper">
            <div class="logo logo-footer">
              logo
            </div>
            <p class="contact__text">one page psd template</p>
          </div>
          <div class="connect">
            <div class="connect__item">
              <img class="connect__icon" src="assets/img/icon1.png" alt="" />
              <p class="connect__text">
                13/2 Elizabeth Street Melbourne VIC 3000, Australia
              </p>
            </div>
            <div class="connect__item">
              <img class="connect__icon" src="assets/img/icon2.png" alt="" />
              <p class="connect__text">
                +(00)1247 457893 mail.example@legngth.com
              </p>
            </div>
          </div>
        </div>
        <div class="copyright">
          <p>Copyright Â© 2019 All Rights Reserved. Designed By Sabbir</p>
        </div>
      </div>
    </footer>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
    <script src="assets/js/app.js"></script>
  </body>
</html>
