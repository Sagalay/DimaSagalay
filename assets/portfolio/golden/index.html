<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="logo_nav clearfix"><img src="img/logo.png" alt="" class="logo">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="title">
				<div class="title_first">
					Welcome To Our Studio!
				</div>
				<h1>
					It’s nice to Meet you
				</h1>
			</div>
			<a href="#" class="button">Tell Me More</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="sec_title">
				<h2>
					Services
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon1.png" alt="Услуга">
					<h3>
						E-Commerce
					</h3>
					<p>
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="services_item">
					<img src="img/icon2.png" alt="Услуга">
					<h3>
						Responsive Web
					</h3>
					<p>
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="services_item">
					<img src="img/icon3.png" alt="Услуга">
					<h3>
						Web Security
					</h3>
					<p>
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.
					</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="sec_title">
				<h2>
					Our portfolio
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="works cleafix">
				<img src="img/img1.jpg" alt="Работа">
				<img src="img/img2.jpg" alt="Работа">
				<img src="img/img3.jpg" alt="Работа">
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="sec_title">
				<h2>
					About Us
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="about_left clearfix">
				<div class="about_item">
				<h4>
					July 2010
				</h4>
				<p class="bold">
					Our Humble Beginnings
				</p>
				<p class="text">
					Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt. Proin 
					iaculis purus consequat sem cure digni 
					ssim. Donec porttitora entum suscipit.
				</p>
				</div>
				<img src="img/about1.jpg" alt="О нас">
			</div>
			<div class="about_right clearfix">
				<img src="img/about2.jpg" alt="О нас">
				<div class="about_item2">
				<h4>
					January 2011
				</h4>
				<p class="bold">
					Facing Startup Battles
				</p>
				<p class="text">
					Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt. Proin 
					iaculis purus consequat sem cure digni 
					ssim. Donec porttitora entum suscipit.
				</p>
				</div>
			</div>
			<div class="third about_left clearfix">
				<div class="about_item">
				<h4>
					December 2012
				</h4>
				<p class="bold">
					Enter The Dark Days
				</p>
				<p class="text">
					Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt. Proin 
					iaculis purus consequat sem cure digni 
					ssim. Donec porttitora entum suscipit.
				</p>
				</div>
				<img src="img/about3.jpg" alt="О нас">
			</div>
			<div class="about_right clearfix">
				<img src="img/about4.jpg" alt="О нас">
				<div class="about_item2">
				<h4>
					february 2014
				</h4>
				<p class="bold">
					Our Triumph
				</p>
				<p class="text">
					Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt. Proin 
					iaculis purus consequat sem cure digni 
					ssim. Donec porttitora entum suscipit.
				</p>
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<p>© Copyright 2014 FreebiesXpress.com</p>
		</div>
	</footer>










<script>document.write('<script src="http://' + (location.host || 'localhost').split(':')[0] + ':35729/livereload.js?snipver=1"></' + 'script>')</script>
</body>
</html>